packages/core/src/components/card/card.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IProps } from "../../common/props";
+
+export interface ICardProps extends IProps {
+    /**
+     * Controls the intensity of the drop shadow beneath the card: the higher
+     * the elevation, the higher the drop shadow. At elevation `0`, no drop
+     * shadow is applied.
+     *
+     * @default 0
+     */
+    elevation?: Elevation;
+
+    /**
+     * Whether the card should respond to user interactions. If set to `true`,
+     * hovering over the card will increase the card's elevation
+     * and change the mouse cursor to a pointer.
+     *
+     * Recommended when `onClick` is also defined.
+     *
+     * @default false
+     */
+    interactive?: boolean;
+
+    /**
+     * Callback invoked when the card is clicked.
+     * Recommended when `interactive` is `true`.
+     */
+    onClick?: (e: React.MouseEvent<HTMLDivElement>) => void;
+}
+
+export enum Elevation {
```

Comment:
this enum feels pretty excessive. I think this should've been a numeric literal union type, `type Elevation  = 0 | 1 | 2 | 3 | 4`. we should make this change in 2.0